duckduckgo/iOS
# Subscription Expired State: 1. Display expired state in settings

Task/Issue URL: https://app.asana.com/0/414235014887631/1207011237405027/f

**Description**:
- Updates settings View to display an expired subscription state

<img width="527" alt="Screenshot 2024-04-16 at 2 25 43 PM" src="https://github.com/duckduckgo/iOS/assets/1156669/a997a663-c06c-4bc4-8325-bea71cc4186d">



**Steps to test this PR**:
1. Build the Alpha Target
2. Make yourself internal user
3. Restore an expired subscription using Email (Or purchase one and wait for it to expire)
4. Reopen Settings
5. Confirm The "Expired view" is shown as in the image above.
